debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -1440,7 +1440,7 @@ private static int resolveSnapshotMaxThreads(Configuration config) {
         return config.getInteger(SNAPSHOT_MAX_THREADS);
     }
 
-    private static ConnectionString resolveConnectionString(Configuration config) {
+    public static ConnectionString resolveConnectionString(Configuration config) {
```

Comment:
I think the behavior is quite general and although from the outsides-outside it is internal, but resolving the conn string from the config is quite a reusable logic? Eventually this could go to a kind of helper class maybe? Especially as sink and source connectors are in the same Maven module/jar. wdyt?

Reply:
So move it into that Unit method, please. Like this it's wrong.

Reply:
see io.debezium.connector.mongodb.shared.SharedMongoDbConnectorConfig

Reply:
done